benches/bench_basic.rs

```diff
@@ -2,18 +2,16 @@
 extern crate criterion;
 use redis;
 
-use futures;
-
 #[path = "../tests/support/mod.rs"]
 mod support;
 
-use futures::{future, stream, Future, Stream};
+use futures::{prelude::*, stream};
```

Comment:
I generally prefer to avoid `*` imports.

Reply:
I agree, but I think it is fine for prelude imports. That is what they are for after all (to my knowledge).

Reply:
I find that it is really easy when modifying a module with explicit `StreamExt,Future, etc` imports that you need to add or remove some which just ends up as noise.

Reply:
I disagree, but it's not up to me anyway. :)